android/core/build.gradle

```diff
@@ -97,11 +97,19 @@ afterEvaluate {
     }
 }
 
+repositories {
```

Comment:
We should not need this.

We can use mavenLocal():
1. cd onnxruntime-native
2. gradle pTML